drape/gl_extensions_list.cpp

```diff
@@ -37,6 +37,20 @@ void GLExtensionsList::Init(dp::ApiVersion apiVersion)
     SetExtension(VertexArrayObject, true);
     SetExtension(UintIndices, true);
   }
+#elif defined(OMIM_OS_LINUX)
+  SetExtension(MapBuffer, true);
+  SetExtension(UintIndices, true);
+  if (apiVersion == dp::ApiVersion::OpenGLES2)
+  {
+    SetExtension(VertexArrayObject, true);
+    SetExtension(MapBuffer, true);
+    SetExtension(MapBufferRange, true);
+  }
+  else // OpenGLES3 branch
```

Comment:
It is OpenGL 2.0, OpenGL 3.0, and OpenGLES3 branch, right?

Reply:
No OpenGL 2.0 is also handled with 2.0 ES in the previous one. In any case I have updated the comment.